### ISSUE TITLE
Naming conflict

### ISSUE THREAD

**Issue by Entomy** [microsoft/L#1]
There's already a language called [L](http://www.nongnu.org/l-lang/), this creates confusion.

**Comment by patrickklaeren** [comment]
> There's already a language called [L](https://github.com/mlemerre/l-lang/), this creates confusion.

I think a lang last updated 8 years ago will survive

**Comment by jeuxjeux20** [comment]
Agreed, an abandoned language won't create much confusion, and Microsoft's L language will gain much more traction. Therefore, the name for the L language LGTM!

**Comment by nukeop** [comment]
> Agreed, an abandoned language won't create much confusion, and Microsoft's L language will gain much more traction. Therefore, the name for the L language LGTM!

It won't gain traction, because it can't be forced onto customers with OEM deals

**Comment by Entomy** [comment]
> I think a lang last updated 8 years ago will survive

I think a multitrillion dollar international company can be a little more creative than constantly replicating community projects and coming up with already existing names.

**Comment by Jay-Madden** [comment]
Its not a community project, its a dead repo with less then 300 commits and zero activity in 8 years. you don't get to claim a single letter name and then sit on it indefinitely.

**Comment by Entomy** [comment]
> Its not a community project, its a dead repo with less then 300 commits and zero activity in 8 years. you don't get to claim a single letter name and then sit on it indefinitely.

Exactly why it should be easy for a company with such vast resources to come up with a name more than a single letter.

**Comment by acikek** [comment]
It stands for Longhorn
